tests/src/Kernel/PermissionEventIntegrationTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
I think we can call it `PermissionEventTest` (as integration is implied by being under `kernel`)